### PR TITLE
rpcclient: add page

### PR DESCRIPTION
**Pull request by dhondta**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/rpcclient.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# rpcclient
+
+> MS-RPC client tool (part of the samba suite).
+> More information: <https://www.samba.org/samba/docs/current/man-html/rpcclient.1.html>.
+
+- Connect to a host:
+
+`rpcclient --user {{domain\username%password}} {{ip}}`
```
Comment (navarroaxel) [inline pages/linux/rpcclient.md:8]:
```suggestion
`rpcclient --user {{domain}}\{{username}}%{{password}} {{ip}}`
```

File: pages/linux/rpcclient.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Connect to a host on a domain without a password:
+
+`rpcclient --user {{username}} --workgroup {{domain}} --no-pass {{ip}}`
+
+- Connect to a host, passing the password hash:
+
+`rpcclient --user {{domain\username}} --pw-nt-hash {{ip}}`
```
Comment (navarroaxel) [inline pages/linux/rpcclient.md:16]:
```suggestion
`rpcclient --user {{domain}}\{{username}} --pw-nt-hash {{ip}}`
```

File: pages/linux/rpcclient.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Enumerate domain users:
+
+`rpcclient $> enumdomusers`
```
Comment (navarroaxel) [inline pages/linux/rpcclient.md:20]: Based on `vim.md` we don't include the prompt:
```suggestion
`enumdomusers`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Just a few comments.

File: pages/linux/rpcclient.md (hunk 3 above)
Comment (dhondta) [reply pages/linux/rpcclient.md:20]: Cfr [`evil-winrm.md`](https://raw.githubusercontent.com/tldr-pages/tldr/master/pages/common/evil-winrm.md), I think it's better to state the context as these commands are only valid in the scope of the sub-terminal once connected using the tool.
What is your opinion ?

File: pages/linux/rpcclient.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Connect to a host, passing the password hash:
+
+`rpcclient --user {{domain}}\{{username}} --pw-nt-hash {{ip}}`
+
+- Enumerate domain users:
+
+`rpcclient $> enumdomusers`
+
+- Enumerate domain groups:
```
Comment (navarroaxel) [inline pages/linux/rpcclient.md:22]: Can we use the verb `display` or `list` to replace `enumerate`?

File: pages/linux/rpcclient.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`rpcclient $> enumdomgroups`
+
+- Enumerate privileges:
+
+`rpcclient $> enumprivs`
+
+- Query specific user information:
```
Comment (navarroaxel) [inline pages/linux/rpcclient.md:30]: Perhaps this will read better?
 ```suggestion
- Display information about a specific user:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

File: pages/linux/rpcclient.md (hunk 4 above)
Comment (dhondta) [reply pages/linux/rpcclient.md]: OK, handled in the latest commit.

File: pages/linux/rpcclient.md (hunk 5 above)
Comment (dhondta) [reply pages/linux/rpcclient.md]: OK, handled in the latest commit.

File: pages/linux/rpcclient.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`rpcclient --user {{username}} --workgroup {{domain}} --no-pass {{ip}}`
+
+- Connect to a host, passing the password hash:
+
+`rpcclient --user {{domain}}\{{username}} --pw-nt-hash {{ip}}`
+
+- Display domain users:
+
+`rpcclient $> enumdomusers`
```
Comment (sbrl) [inline pages/linux/rpcclient.md:20]: Can we wrap this in a call to the `rpcclient` command at all? For example, to execute php code directly you can do `php -r '{{insert_code_here}}'`

Comment (dhondta) [reply pages/linux/rpcclient.md]: OK, please see in my last commit ; I added one example with remote shell commands in the command line but I think it's good to keep the examples with the `rpcclient` shell as it is the preferred solution when working with the tool.

Comment (sbrl) [reply pages/linux/rpcclient.md]: I see. In other pages though, we  haven't used a shell prefix.

/cc @owenvoke / @mebeim / @agnivade for a 2nd opinion.

Comment (dhondta) [reply pages/linux/rpcclient.md]: @sbrl I did it previously with [`evil-winrm.md`](https://raw.githubusercontent.com/tldr-pages/tldr/master/pages/common/evil-winrm.md). Isn't it compliant with your guidelines ?

Comment (sbrl) [reply pages/linux/rpcclient.md]: Ah, thanks for that  @dhondta! It's not that it doesn't match the guidelines, it's just that I'm always unsure with subshell examples. If it's  been done before, that's good enough for me :slightly_smiling_face:

**Review by sbrl — APPROVED** [review]
Thanks, @dhondta!

File: pages/linux/rpcclient.md (hunk 6 above)
Comment (agnivade) [reply pages/linux/rpcclient.md]: Even pages like vim has internal commands. It's okay.

**Pull request merged**